core/src/main/scripts/importer/validateStudies.py

```diff
@@ -0,0 +1,185 @@
+            var_traceback = traceback.format_exc()
+            print '\x1b[31m' + "Error occured during validation:"
+            print var_traceback + '\x1b[0m'
+            validation_exit_status = 1  # Here also set invalid validation exit status for failing circleCI
+            # Reset logger handlers to suppress stdout output, will be written to log file
+            validator_logger = logging.getLogger(validateData.__name__)
+            validator_logger.handlers = []
+
+    return validation_exit_status
+
+
+def interface(args=None):
+    """Parse arguments from input"""
+
+    parser = argparse.ArgumentParser(description='Wrapper where cBioPortal study validator is run for multiple studies')
+    parser.add_argument('-d', '--root-directory',
+                        type=str, help='Path to directory with all studies that should be validated')
+    parser.add_argument('-l', '--list-of-studies',
+                        type=str, help='List with paths of studies which should be validated')
+    parser.add_argument('-html', '--html-folder',
+                        type=str, help='Path to folder for output HTML reports')
+    # Only one of the portal arguments can be given, therefore set as mutually exclusive group
+    portal_mode_group = parser.add_mutually_exclusive_group()
+    portal_mode_group.add_argument('-u', '--url_server',
+                                   type=str,
+                                   default='http://localhost:8080/cbioportal',
+                                   help='URL to cBioPortal server. You can '
+                                        'set this if your URL is not '
+                                        'http://localhost:8080/cbioportal')
+    portal_mode_group.add_argument('-p', '--portal_info_dir',
+                                   type=str,
+                                   help='Path to a directory of cBioPortal '
+                                        'info files to be used instead of '
+                                        'contacting a server')
+    portal_mode_group.add_argument('-n', '--no_portal_checks',
+                                   action='store_true',
+                                   help='Skip tests requiring information '
+                                        'from the cBioPortal installation')
+    parser.add_argument('-P', '--portal_properties', type=str,
+                        help='portal.properties file path (default: assumed hg19)',
+                        required=False)
+
+    args = parser.parse_args(args)
+
+    # Check if -d or -l was given as input, otherwise let the parser give an error and stop
+    if not (args.root_directory or args.list_of_studies):
+        parser.error('No action requested, add -d or -l')
+
+    return args
+
+
+if __name__ == '__main__':
+    parsed_args = interface()
+    exit_status = main(parsed_args)
+
+    print "\n\nOverall exit status: %s" %exit_status
```

Comment:
you still need to use this to let the script exit with 1 if there is any error. I.e. add:
```
sys.exit(exit_status)
```